developer_manual/app/schema.rst

```diff
@@ -1,3 +1,101 @@
+===================
+Database Migrations
+===================
+
+.. sectionauthor:: Thomas Müller <thomas.mueller@tmit.eu>
+
+ownCloud uses `Doctrine Migrations <http://www.doctrine-project.org/projects/migrations.html>`_ to manage any kind of database changes.
+
+Any apps which wants to use this mechanism has to enable it in :file:`appinfo/info.xml` by adding
```

Comment:
Any apps which **want** to use this mechanism **have** to enable it in
or
Any **app** which wants to use this mechanism has to enable it in